xml/deployment_installation.xml

```diff
@@ -825,14 +825,350 @@
+admin_flavor: m1.large
+&hellip;
+         </screen>
+       </warning>
+
+        <informalfigure>
+         <mediaobject>
+          <imageobject role="fo">
+           <imagedata fileref="horizon_launch_stack.png" width="100%"/>
+          </imageobject>
+          <imageobject role="html">
+           <imagedata fileref="horizon_launch_stack.png" width="100%"/>
+          </imageobject>
+         </mediaobject>
+        </informalfigure>
+       </step>
+       
+      <step>
+       <para>
+        Click <guimenu>Next</guimenu>.
+       </para>
+      </step>
+
+       <step>
+        <para>
+         Now you need to define more information about the stack.
+        </para>
+        <variablelist>
+         <varlistentry>
+          <term><literal>Stack Name</literal>
+          </term>
+          <listitem>
+           <para>
+            Give your stack a name
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>Password</literal>
+          </term>
+          <listitem>
+           <para>
+            Your &soc; password
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>Image</literal>
+          </term>
+          <listitem>
+           <para>
+            Select the image your machines will be created from
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>root_password</literal>
+          </term>
+          <listitem>
+           <para>
+            Set the root password for your cluster machines
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>admin</literal>/master/worker_flavor</term>
+          <listitem>
+           <para>
+            Select the machine flavor for your nodes
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>worker_count</literal>
+          </term>
+          <listitem>
+           <para>
+            Number of worker nodes to be launched
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>external_net</literal>
+          </term>
+          <listitem>
+           <para>
+            Select an external network that your cluster will be reachable from
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>internal_net_cidr</literal>
+          </term>
+          <listitem>
+           <para>
+            The internal network range to be used inside the cluster
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>dns_nameserver</literal>
+          </term>
+          <listitem>
+           <para>
+            Internal nameserver for the cluster
```

Comment:
'name server', not 'nameserver'. Refer to the [terminology in the Style Guide](http://docserv.nue.suse.com/documents/Style_Guide/docu_styleguide/html/app.terminology.html#sec.terminology)